### PR TITLE
add structure

### PR DESCRIPTION
**Pull request by AlexeyDunnikov**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  })
+);
+
+const PORT = process.env.PORT || 3000;
+
+app.listen(PORT, () => {
+  console.log(`Server is running on port ${PORT}`);
+});
+
+function normalizeArr(arr) {
```
Comment (vitali-working) [inline app.js]: Why don't you use `normalizeArr` from utils?

File: controllers/categories_controller.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+const normalizeArr = require("../utils/normalize_arr");
+
+function CategoriesController(connection) {
```
Comment (vitali-working) [inline controllers/categories_controller.js]: param `connection` is not used

File: controllers/categories_controller.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  controllerMethods = {};
```
Comment (vitali-working) [inline controllers/categories_controller.js]: `let` or `const`?

File: models/cars_model.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
+function CarsModel(connection) {
+  modelMethods = {};
```
Comment (vitali-working) [inline models/cars_model.js]: `let` or `const`?

File: models/cars_model.js [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+  modelMethods.getCarsInfo = function (req, res) {
```
Comment (vitali-working) [inline models/cars_model.js]: `getCarsInfo` -> `getAllCars`?

File: models/categories_model.js [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+      connection.query(
+        `SELECT * FROM parts_subcategories`,
+        (err, result, fields) => {
+          if (err) reject(err);
+          resolve(result);
+        }
+      );
+    });
+
+    return Promise.all([categories, subcategories]).then(
```
Comment (vitali-working) [inline models/categories_model.js]: please, add `.catch` for error handling

File: public/js/index.js [hunk 7]
```diff
@@ -0,0 +1 @@
+console.log('hello');
```
Comment (vitali-working) [inline public/js/index.js]: remove it)

File: utils/normalize_arr.js [hunk 8]
```diff
@@ -0,0 +1,13 @@
+function normalizeArr(arr) {
+  const res = {};
+
+  arr = JSON.parse(JSON.stringify(arr));
```
Comment (vitali-working) [inline utils/normalize_arr.js]: why do you need this transformation?

File: controllers/categories_controller.js (hunk 2 above)
Comment (AlexeyDunnikov) [reply controllers/categories_controller.js]: In the future it will be used, when i'll add model

Comment (vitali-working) [reply controllers/categories_controller.js]: remove it for now.

File: controllers/catalogController.js [hunk 9]
```diff
@@ -52,5 +52,50 @@ module.exports = (connection) => {
...
+    const brands = await catalogModel.getBrandsBySubcatId(subcatId);
+
+    const minPrice = await catalogModel.getMinPriceBySubcatId(subcatId);
+
+    const maxPrice = await catalogModel.getMaxPriceBySubcatId(subcatId);
+
+   //console.log(parts);
+
+    const options = {
+      title: 'Каталог',
```
Comment (vitali-working) [inline controllers/catalogController.js]: 'Каталог' - move to constant

Comment (AlexeyDunnikov) [reply controllers/catalogController.js]: Such as 
const TITLES = {
    MAIN: "Главная",
    CATALOG: "Каталог",
    ABOUT: "О компании",
  };
?

Comment (vitali-working) [reply controllers/catalogController.js]: yes

**Pull request merged**